BrowserKit/Sources/Common/Constants/StandardImageIdentifiers.swift

```diff
@@ -8,6 +8,11 @@ import Foundation
 /// When adding new identifiers, please respect alphabetical order.
 /// Sing the song if you must.
 public struct StandardImageIdentifiers {
+    // Icon size 8x8
+    public struct Tiny {
+        public static let chevronDown = "chevronDownTiny"
```

Comment:
This icon does not exist in the acorn repo and therefor shouldn't be added here.

Reply:
Apparently, it does in the acorn repo in `.svg` format: https://github.com/FirefoxUX/acorn-icons/blob/main/icons/mobile/8/svg/chevron-down-8.svg.

Reply:
It would be handy if there was some indication of this in the file documentation, I wouldn't have known this either. 😅

Reply:
@ih-codes, before opening the PR I tried using `chevronDownLarge` and it didn't look but it didn't look like the one we have in Figma. `chevronDownLarge` has a padding and when we size it to `8p`, the arrow looks smaller than it should be. I encourage you to take a look as well, maybe I could be wrong.

Reply:
You're totally right @PARAIPAN9. It definitely has excess padding. 😔 

<img width="78" alt="Screenshot 2024-10-07 at 3 21 54 PM" src="https://github.com/user-attachments/assets/c24072b6-b4f5-4df7-92cb-eac8515ad023">

It looks like all the Acorn images have a baked in (unspecified?) amount of padding, but our design calls for 0 padding. I'm guessing we should simply add a PDF version of the same resource you originally added @PARAIPAN9. Is that reasonable @thatswinnie?

Reference:
<img width="348" alt="Screenshot 2024-10-07 at 3 26 52 PM" src="https://github.com/user-attachments/assets/c6425b24-6c0e-4642-a12c-f01b254b9469">

Reply:
Before adding a pdf I suggest to talk to @cwzilla and Aarjav. We should use standardizes images as much as we can. When I updated all the existing images to use Acorn images instead, there was multiple conversations with Crystal about images that had excess padding and we got that removed in certain cases. So please reach out to Crystal and Aarjav on Slack to figure this out.

Reply:
Thanks @thatswinnie!!

Reply:
Thank you both!! 🙏